packages/protoplugin/src/error.ts

```diff
@@ -32,3 +34,13 @@ export function reasonToString(reason: unknown): string {
   }
   return String(reason);
 }
+
+export function isPluginOptionError(arg: unknown): arg is PluginOptionError {
+  if (!(arg instanceof Error)) {
+    return false;
+  }
+  if (Object.getPrototypeOf(arg) === PluginOptionError.prototype) {
+    return true;
+  }
```

Comment:
This is equivalent to `instanceof PluginOptionError`, so we cannot use it.

It's fine to simply check for `instanceof Error` and `name === "PluginOptionError"`.